scripts/qa_crowbarsetup.sh

```diff
@@ -1866,10 +1894,18 @@ function custom_configuration()
                 fi
             fi
 
+            #FIXME: till bug#930986 this part can be enabled once again
+            # https://bugzilla.suse.com/show_bug.cgi?id=930986
+            #
             # assign neutron-network role to one of SLE12 nodes
-            if [ -n "$want_sles12" ] && [ -z "$hacloud"] && [ -n "$want_neutronsles12" ] && iscloudver 5plus ; then
+            #if [ -n "$want_sles12" ] && [ -z "$hacloud"] && [ -n "$want_neutronsles12" ] && iscloudver 5plus ; then
                 # 2015-03-03 off-by-default because Failed to validate proposal: Role neutron-network can't be used for suse 12.0, windows /.*/ platform(s).
-                proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-network']" "['$sle12node']"
+            #   proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-network']" "['$sle12node']"
+            #fi
+
+            if [ -n "$want_sles12_controller" ] ; then
+                proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-server']" "['$sle12controller']"
+                proposal_set_value neutron default "['deployment']['neutron']['elements']['neutron-network']" "['$sle12controller']"
```

Comment:
Right, this part should be commented, not the previous one. ``neutron-network`` role is fine, it was already working before. ``neutron-server`` needs to be disallowed for now